daemon/mgr/container.go

```diff
@@ -248,9 +252,12 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		return nil, fmt.Errorf("NetworkingConfig cannot be nil")
 	}
 
-	id, err := mgr.generateID()
-	if err != nil {
-		return nil, err
+	id := ""
+	if id = config.ContainerID; id == "" {
```

Comment:
Why not code like this:
```go
id := config.ContainerID
if id == "" {
...
}
```
It seems more concise. 😄

Reply:
Done.